developers_chamber/scripts/project.py

```diff
@@ -100,20 +110,34 @@ def exec_command(ctx, project_name, compose_file, container, command):
 @click.option('--project-name', '-p', help='Name of the project', type=str, required=True, default=default_project_name)
 @click.option('--compose-file', '-f', help='Compose file', type=str, required=True, multiple=True,
               default=default_compose_files)
-@click.option('--container', '-c', help='Container name', type=str, required=False, multiple=True)
-def up(project_name, compose_file, container):
+@click.option('--container', '-c', help='Container name', type=str, required=False, multiple=True,
+              default=default_up_containers)
+@click.option('--all', '-a', 'all_containers', help='Run all container in compose file', default=False, is_flag=True)
```

Comment:
helper to run all containers (because default value of container attribute can be annoying)